docs/dataservice-read-catalog-example.md

```diff
@@ -44,241 +51,329 @@ After building and running the example, you see the following:
 [INFO] read-example - Request partition data - Success, data size - 3375
 ```
 
-## Building and running on Android
+## Build and Run on Android
 
-This example shows how to integrate the HERE OLP SDK for C++ libraries into the Android project and read data from a catalog layer and partition.
+To integrate the HERE OLP SDK for C++ libraries in the Android project:
 
-### Prerequisites
+- [Set up prerequisites](#prerequisites-android)
+- [Build the HERE OLP SDK for C++](#build-sdk-android)
+- [Build and Run the APK](#build-and-run-android)
 
-* Set up the Android environment.
-* Replace the placeholders in `examples/dataservice-read/example.cpp` with your app access key id and secret access key.
+### <a name="prerequisites-android"></a>Prerequisites
 
-### Build HERE OLP SDK for C++
+Before you integrate the HERE OLP SDK for C++ libraries in your Android project:
```

Comment:
```suggestion
Before you integrate the HERE OLP SDK for C++ libraries in Android example project:
```
This section describes how to use our Android example project, not custom user's project

Reply:
Thank you!
Fixed